presto-i18n-functions/src/main/java/com/facebook/presto/i18n/functions/I18nMyanmarFunctions.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.i18n.functions;
+
+import com.facebook.presto.spi.function.Description;
+import com.facebook.presto.spi.function.ScalarFunction;
+import com.facebook.presto.spi.function.SqlType;
+import com.facebook.presto.spi.type.StandardTypes;
+import com.google.myanmartools.TransliterateZ2U;
+import com.google.myanmartools.ZawgyiDetector;
+import io.airlift.slice.Slice;
+
+import static io.airlift.slice.Slices.utf8Slice;
+
+public final class I18nMyanmarFunctions
+{
+    private I18nMyanmarFunctions() {}
+
+    static ZawgyiDetector detector = new ZawgyiDetector();
+    static TransliterateZ2U z2uTransliterator = new TransliterateZ2U("Zawgyi to Unicode");
```

Comment:
Actually should these also be `private static final`? Put the variables before the constructor per convention.

Reply:
They can be - updated & shifted the definition of these and ZAWGYI_PROBABILITY_THRESHOLD